### PR TITLE
added selectLayersByType method

### PR DESCRIPTION
**Pull request by FinnFrotscher**
`nexrender.selectLayersByType` invokes callback for each layer of a certain type.
needed to manipulate the Font Family of every text layer in my compositions.

### CODE REVIEW

**Comment by inlife** [comment]
@FinnFrotscher thank you for the contribution!
published as `v1.13.0`

**Comment by FinnFrotscher** [comment]
love your work!

**Comment by FinnFrotscher** [comment]
do you think it would be useful to extend the functionality of this function to include a name filter?

File: packages/nexrender-core/src/assets/nexrender.jsx [hunk 1]
```diff
@@ -95,6 +95,50 @@ nexrender.selectLayersByName = function(compositionName, name, callback, types)
...
+
+    /* step 1: collect all matching layers */
+    for (var j = 1; j <= comp.numLayers; j++) {
+      var layer = comp.layer(j);
+
+      if (!(layer instanceof type)) continue;
+
+      if (nexrender.typesMatch(types, layer)) {
+        foundOnce = true;
+        items.push(layer);
```
Comment (FinnFrotscher) [inline packages/nexrender-core/src/assets/nexrender.jsx:121]:
```
if (layerName){
  if (layerName == layer.name) {
    items.push(layer)
    foundOnce = true
  }
} else {
  items.push(layer) 
  foundOnce = true
}
```

File: packages/nexrender-core/src/assets/nexrender.jsx [hunk 2]
```diff
@@ -95,6 +95,50 @@ nexrender.selectLayersByName = function(compositionName, name, callback, types)
     }
 };
 
+/* call callback for every layer matching specific type and composition */
+nexrender.selectLayersByType = function(
+  compositionName,
```
[the remark this answers is not in the document]
Comment (FinnFrotscher) [reply packages/nexrender-core/src/assets/nexrender.jsx:100]: @inlife

**Comment by FinnFrotscher** [comment]
@inlife

**Comment by inlife** [comment]
Hey @FinnFrotscher 
I'm not sure I understand how that filtering would work? What is it needed for?

**Pull request merged**